scripts/build_release.sh

```diff
@@ -71,7 +60,6 @@ fi
 echo ${BUILD_NUMBER} > ./buildnumber.dat
 git add -A
 git commit -m "Build ${BUILD_NUMBER}"
-git push
```

Comment:
I dont see any `git push` for this commit.  As I understand it, `git push --tags` doesn't push commits.

Reply:
in build_release_upload.sh `git push origin --tags ${BRANCH}` achieves both the tags and the branch commits

Reply:
updated that...